spec/controllers/admin/cms/collections_controller_spec.rb

```diff
@@ -1,5 +1,163 @@
 require 'rails_helper'
 
-RSpec.describe Admin::CollectionsController, type: :controller do
+RSpec.describe Admin::Cms::CollectionsController, type: :controller do
+  let(:vendor) { Vendor.create!(uid: 'voice_base', name: 'VoiceBase') }
+  let(:collection) do
+    Collection.create!(
+      description: "A summary of the collection's content",
+      url: "collection_catalogue_reference",
+      uid: "collection-uid",
+      title: "The collection's title",
+      call_number: "catalogue_reference",
+      vendor: vendor
+    )
+  end
 
+  describe "GET #show" do
+    let(:action) { get :show, id: collection.uid }
+
+    it "is successful" do
+      action
+      expect(response.code).to eq("200")
+    end
+
+    it "displays the collection with related items" do
+      action
+      expect(response).to render_template(:show)
+    end
+  end
+
+  describe "GET #new" do
+    let(:action) { get :new }
+
+    it "is successful" do
+      action
+      expect(response.code).to eq("200")
+    end
+
+    it "displays the form to create a new collection" do
+      action
+      expect(response).to render_template(:new)
+    end
+  end
+
+  describe "POST #create" do
+    context "valid request" do
+      let(:params) do
+        {
+          uid: "new_collection",
+          title: "New collection title",
+          call_number: "New catalogue reference",
+          description: "New description of collection",
+          url: "new_collection_catalogue_reference",
+          image: File.open(Rails.root.join('spec', 'fixtures', 'image.jpg')),
+          vendor_id: vendor.id
+        }
+      end
+      let(:action) { post :create, collection: params }
```

Comment:
I can confirm rspec-rails version 3.4.2 included in Gemfile.lock and the latest version is 3.6.0. It fails if I attempt to use the new syntax, thank you for pointing that out 👍